cmd/migrate/v3/migrate.go

```diff
@@ -0,0 +1,89 @@
+package v3
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"gopkg.in/yaml.v3"
+
+	v2 "github.com/forbole/juno/v3/cmd/migrate/v2"
+	"github.com/forbole/juno/v3/cmd/parse"
+	"github.com/forbole/juno/v3/database"
+	databaseconfig "github.com/forbole/juno/v3/database/config"
+	v3db "github.com/forbole/juno/v3/database/legacy/v3"
+	"github.com/forbole/juno/v3/database/postgresql"
+	"github.com/forbole/juno/v3/types/config"
+)
+
+// RunMigration runs the migrations from v2 to v3
+func RunMigration(parseConfig *parse.Config) error {
+	// Migrate the config
+	cfg, err := migrateConfig()
+	if err != nil {
+		return fmt.Errorf("error while migrating config: %s", err)
+	}
+
+	bz, err := yaml.Marshal(&cfg)
+	if err != nil {
+		return fmt.Errorf("error while serializing config: %s", err)
+	}
+
+	err = ioutil.WriteFile(config.GetConfigFilePath(), bz, 0666)
+	if err != nil {
+		return fmt.Errorf("error while writing v3 config: %s", err)
+	}
+
+	// Migrate the database
+	err = migrateDb(cfg, parseConfig)
+	if err != nil {
+		return fmt.Errorf("error while migrating database: %s", err)
+	}
+
+	return nil
+}
+
+func migrateConfig() (Config, error) {
+	cfg, err := v2.GetConfig()
+	if err != nil {
+		return Config{}, fmt.Errorf("error while reading v2 config: %s", err)
+	}
+
+	return Config{
+		Node:  cfg.Node,
+		Chain: cfg.Chain,
+		Database: databaseconfig.Config{
+			Name:               cfg.Database.Name,
+			Host:               cfg.Database.Host,
+			Port:               cfg.Database.Port,
+			User:               cfg.Database.User,
+			Password:           cfg.Database.Password,
+			SSLMode:            cfg.Database.SSLMode,
+			Schema:             cfg.Database.Schema,
+			MaxOpenConnections: cfg.Database.MaxOpenConnections,
+			MaxIdleConnections: cfg.Database.MaxIdleConnections,
+			PartitionSize:      0,
```

Comment:
This is causing `partition_size` and `partition_batch` values to be changed to 0 inside the config.yaml file and the migration always fails